app/views/summary/index.html.haml

```diff
@@ -0,0 +1,43 @@
+%div.inset
+  %table.table.table-striped.table-bordered.table-responsive
+    %caption
+      %span.namespace= @summary.client_namespace
+      Requests: Fiscal Year
+      = @summary.fiscal_year
```

Comment:
slack conversation: https://18f.slack.com/archives/cap/p1449155873001851

seems like it is :+1: for keeping this feature, and I have added tests.